ray-operator/controllers/ray/utils/util.go

```diff
@@ -647,3 +645,42 @@ func GetRayClusterNameFromService(svc *corev1.Service) string {
 	}
 	return svc.Spec.Selector[RayClusterLabelKey]
 }
+
+// getDiscoveryClient returns a discovery client for the current reconciler
+func getDiscoveryClient(config *rest.Config) (*discovery.DiscoveryClient, error) {
```

Comment:
This function just passes the variable to another function. It's not a good pattern, especially since it doesn't improve readability. We can remove this function.